test/e2e/serial/config/config.go

```diff
@@ -67,3 +76,64 @@ func GetRteCiImage() string {
 	}
 	return nropTestCIImage
 }
+
+func CheckNodesTopology(ctx context.Context) error {
+	kconfigs, err := validator.GetKubeletConfigurationsFromTASEnabledNodes(ctx, e2eclient.Client)
+	if err != nil {
+		return fmt.Errorf("error while trying to get KubeletConfigurations. error: %w", err)
+	}
+
+	var nrtList nrtv1alpha2.NodeResourceTopologyList
+
+	err = e2eclient.Client.List(context.TODO(), &nrtList)
+	if err != nil {
+		return fmt.Errorf("error while trying to get NodeResourceTopology objects. error: %w", err)
+	}
+
+	policies := []nrtv1alpha2.TopologyManagerPolicy{
+		nrtv1alpha2.SingleNUMANodeContainerLevel,
+		nrtv1alpha2.SingleNUMANodePodLevel,
+	}
+	nrts := e2enrt.FilterByPolicies(nrtList.Items, policies)
+	singleNumaNodeNumber := len(nrts)
+	if singleNumaNodeNumber < minNumberOfNodesWithSameTopology {
+		return fmt.Errorf("not enough nodes with valid policy - found %d, need at least %d", len(nrts), minNumberOfNodesWithSameTopology)
```

Comment:
this can be removed now that the accurate check is done in L109

Reply:
Let's do this on another PR to refactor/cleanup all validation code, ok?

Reply:
if this check we're adding here is superseded by another test done later in this very function being added by this PR, it seems simpler to just remove this redundant check, though.

Reply:
Sorry my bad, I readed it in a hurry and thought we were talking about the checks in the tests cases .... 

yes, this is deleted in the last version.